api/src/pcapi/scripts/offerer/delete_cascade_venue_by_id.py

```diff
@@ -64,8 +74,26 @@ def delete_cascade_venue_by_id(venue_id: int) -> None:
     ).delete(synchronize_session=False)
 
     offer_ids_to_delete = [id[0] for id in db.session.query(Offer.id).filter(Offer.venueId == venue_id).all()]
+    collective_offer_ids_to_delete = [
```

Comment:
[facultatif] pourquoi ne pas faire de vrai boucles for plutot qu'une comprehension list sur 6 lignes ?